index.html

```diff
@@ -6,59 +6,244 @@
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="./assets/project3.png" alt="image work">
+                </div>
+                <h2 class="card-title">
+									Facebook 360
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="./assets/project1.png" alt="image work">
+                </div>
+                <h2 class="card-title">
+                     Uber Navigation
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="./assets/project4.png" alt="image work">
+                </div>
+                <h2 class="card-title">
+                    Tonic
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                    See Project
+                </button>
+            </article>
+            <article class="card-work">
+                <div class="card-image">
+                    <img src="./assets/project2.png" alt="image work">
+                </div>
+                <h2 class="card-title">
+                    Multi-Post Stories Gain+Glory
+                </h2>
+                <ul class="card-tags">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <button type="button">
+                    See Project
+                </button>
+            </article>
+        </div>
+    </section>
+    <section class="container-flex about-myself">
+        <div class="container">
+            <h1>About me</h1>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you
```

Comment:
- [ ] Please it is not a good practise to use more than one h1 tag in the document 
use h2-h6 instead so your code became more symantic